packages/ai/poetry.lock

```diff
@@ -19,8 +19,8 @@ category = "dev"
 description = "Classes Without Boilerplate"
 name = "attrs"
 optional = false
-python-versions = "*"
-version = "18.2.0"
+python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*"
```

Comment:
My vote would be to chop 2.7 from this list altogether. Let's just support 3 only. It will reduce complexity and 2 is completely retired in 9 months.

Reply:
This is an autogenerated file, but I agree, in the case of things that we can control, we should only support version 3 and higher.